kiwi/mount_manager.py

```diff
@@ -148,10 +155,13 @@ def umount(self, raise_on_busy: bool = True) -> bool:
                     Command.run(['umount', self.mountpoint])
                     umounted_successfully = True
                     break
-                except Exception:
+                # we only want to catch KiwiCommandError, everything else
+                # indicates either a bug in the code or a bigger problem, like
+                # umount not being available
+                except KiwiCommandError as kw_err:
                     log.warning(
-                        '%d umount of %s failed, try again in 1sec',
-                        busy, self.mountpoint
+                        '%d umount of %s failed with %s, try again in 1sec',
```

Comment:
As you are on it, can you also change the message to either use f-string or format()